src/algebra/big_operators/basic.lean

```diff
@@ -634,6 +634,15 @@ lemma prod_range_succ' (f : ℕ → β) :
 | (n + 1) := by rw [prod_range_succ (λ m, f (nat.succ m)), mul_assoc, ← prod_range_succ'];
                  exact prod_range_succ _ _
 
+lemma prod_range_add (f : ℕ → β) (n : ℕ) (m : ℕ) :
```

Comment:
To be clear - does this work?

```suggestion
@[to_additive]
lemma prod_range_add (f : ℕ → β) (n : ℕ) (m : ℕ) :
```

Reply:
I get the error
```
basic.lean:638:0
type mismatch at application
  0
term
  nat.has_one
has type
  has_one ℕ
but is expected to have type
  has_zero ℕ
```

Reply:
Yeah, I've seen that before. Don't worry about it then.